test/demo/createToast.inttest.ts

```diff
@@ -0,0 +1,95 @@
+import 'jest';
+
+import {Application, Window} from 'hadouken-js-adapter';
+
+import {NotificationOptions, Notification} from '../../src/client';
+
+import * as notifsRemote from './utils/notificationsRemoteExecution';
+import {isCenterShowing, getCenterCardsByNotification} from './utils/notificationCenterUtils';
+import {delay} from './utils/delay';
+import {getToastWindow, getToastCards} from './utils/toastUtils';
+import {createApp} from './utils/spawnRemote';
+import {assertNotificationStored} from './utils/storageRemote';
+import {assertDOMMatches, CardType} from './utils/noteCardUtils';
+
+const options: NotificationOptions = {
+    body: 'Test Notification Body',
+    title: 'Test Notification Title'
+};
+
+const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+
+describe('When calling createNotification with the notification center not showing', () => {
+    let testApp: Application;
+    let testWindow: Window;
+
+    let createPromise: Promise<Notification>;
+    let note: Notification;
+
+    beforeAll(async () => {
+        // Toggle the center on/off based on test type
+        if (await isCenterShowing()) {
+            await notifsRemote.toggleNotificationCenter(testManagerIdentity);
+        }
+    });
+
+    beforeEach(async () => {
+        testApp = await createApp(testManagerIdentity, {});
+        testWindow = await testApp.getWindow();
+
+        createPromise = notifsRemote.create(testWindow.identity, options);
+
+        // We want to be sure the operation is completed, but don't care if it succeeds
+        note = await createPromise.catch(() => ({} as Notification));
+    });
+
+    afterEach(async () => {
+        await notifsRemote.clearAll(testWindow.identity);
+        await testApp.quit();
+    });
+
+    test('The promise resolves to the fully hydrated notification object', async () => {
+        await expect(createPromise).resolves;
+        expect(note).toMatchObject(options);
+    });
+
+    test('A toast is shown for the notification', async () => {
+        // The toast creation is not awaited as part of notes.create, so we add a small
+        // delay to allow the window time to spawn.
+        await delay(500);
```

Comment:
We're being inconsistent with how long we're waiting. This is given 100ms in createCenter. Suggest make consistent and extract as constant